libs/extensions/angular/image-banner/src/image-banner.component.html

```diff
@@ -48,47 +28,63 @@
               <ng-content select="[bodyText]"></ng-content>
             }
           </p>
-
-          @if (!showButtonInNarrowView && externalLink) {
-            <div class="main-content-body-action-link">
-              <kirby-icon name="link"></kirby-icon>
-            </div>
-          }
         </div>
       </div>
-      <!-- On large screens we also show a button-like anchor tag in addition to the entire banner anchor -->
       @if (externalLink) {
+        <!-- Only one of below anchors is displayed at a time -
+            we want different behavior on varying screen widths -->
         <a
           kirby-button
           class="main-content-body-action-text"
```

Comment:
Rename class and add class wide-view-action
```suggestion
          class="main-content-action wide-view-action"
```